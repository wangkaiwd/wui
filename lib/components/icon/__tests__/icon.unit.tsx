import React from 'react';
import Icon from '../icon';
// 官方文档地址：https://zh-hans.reactjs.org/docs/test-renderer.html
import render from 'react-test-renderer';

describe('Icon组件', () => {
  it('render successfully', () => {
    const json = render.create(<Icon name={'qq'}/>).toJSON();
    expect(json).toMatchSnapshot();
  });
});
