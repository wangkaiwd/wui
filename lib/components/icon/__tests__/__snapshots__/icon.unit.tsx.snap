// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon组件 render successfully 1`] = `
<svg
  className="wui-icon"
>
  <use
    xlinkHref="#qq"
  />
</svg>
`;
